src/main/java/org/sagebionetworks/web/shared/users/AclUtils.java

```diff
@@ -19,35 +33,31 @@
 	// because the elements are all ENUMS.
 	static {
 		permToACCESS_TYPE = new HashMap<PermissionLevel, Set<ACCESS_TYPE>>();
-		TreeSet<ACCESS_TYPE> accessList = new TreeSet<ACCESS_TYPE>();
-
-		accessList.add(ACCESS_TYPE.READ);
-		permToACCESS_TYPE.put(PermissionLevel.CAN_VIEW, new TreeSet<ACCESS_TYPE>(accessList));
-
-		//Score in Evaluation (must clean up new access_type, does not modify existing permission levels)
-		permToACCESS_TYPE.put(PermissionLevel.CAN_SCORE_EVALUATION, addTypes(accessList, ACCESS_TYPE.READ_PRIVATE_SUBMISSION, ACCESS_TYPE.UPDATE_SUBMISSION));
-				
-		//Participate in Evaluation (must clean up new access_type, does not modify existing permission levels)
-		permToACCESS_TYPE.put(PermissionLevel.CAN_PARTICIPATE_EVALUATION, addTypes(accessList, ACCESS_TYPE.PARTICIPATE, ACCESS_TYPE.SUBMIT));
-		
-		accessList.add(ACCESS_TYPE.UPDATE);
-		
-		accessList.add(ACCESS_TYPE.CREATE);
-		permToACCESS_TYPE.put(PermissionLevel.CAN_EDIT, new TreeSet<ACCESS_TYPE>(accessList));
-		
-		accessList.add(ACCESS_TYPE.DELETE);
-		permToACCESS_TYPE.put(PermissionLevel.CAN_EDIT_DELETE, new TreeSet<ACCESS_TYPE>(accessList));
 		
-		accessList.add(ACCESS_TYPE.CHANGE_PERMISSIONS);
-		accessList.add(ACCESS_TYPE.CHANGE_SETTINGS);
-		permToACCESS_TYPE.put(PermissionLevel.CAN_ADMINISTER, new TreeSet<ACCESS_TYPE>(accessList));
+		permToACCESS_TYPE.put(PermissionLevel.CAN_VIEW, new TreeSet<ACCESS_TYPE>(Arrays.asList(
+				READ)));
+		permToACCESS_TYPE.put(PermissionLevel.CAN_EDIT, new TreeSet<ACCESS_TYPE>(Arrays.asList(
+				CREATE, READ, UPDATE)));
+		permToACCESS_TYPE.put(PermissionLevel.CAN_EDIT_DELETE, new TreeSet<ACCESS_TYPE>(Arrays.asList(
+				CREATE, READ, UPDATE, DELETE)));		
+		// TODO replace this list with ModelConstants.ENITY_ADMIN_ACCESS_PERMISSIONS
+		permToACCESS_TYPE.put(PermissionLevel.CAN_ADMINISTER, new TreeSet<ACCESS_TYPE>(Arrays.asList(
+				CREATE, READ, UPDATE, DELETE, CHANGE_PERMISSIONS, CHANGE_SETTINGS)));
+		// TODO replace this list with ModelConstants.ENITY_ADMIN_ACCESS_PERMISSIONS
+		permToACCESS_TYPE.put(PermissionLevel.OWNER, new TreeSet<ACCESS_TYPE>(Arrays.asList(
```

Comment:
perhaps we should remove the "Owner" item from the access levels combo box completely (and the owner permission level/bundle)